src/app/static/src/tests/components/downloadResults/uploadModal.test.ts

```diff
@@ -298,6 +287,14 @@ describe(`uploadModal `, () => {
         const modal = wrapper.find(".modal");
         expect(modal.classes()).toContain("show");
 
+        //Refresh adrUpload upload files to trigger watch, which populates uploadFilesToADR
+        store.state.adrUpload.uploadFiles= {...fakeMetadata};
+
+        //Click ok to trigger population of uploadFilesPayload
+        const okBtn = modal.find("button.btn-red");
+        expect(okBtn.element)
+        await okBtn.trigger("click");
+
```

Comment:
This was the main reason why downloadIsReady wasn't ever returning true - it relies on `uploadFilesPayload` to be set, which in turn derived from `uploadFilesToADR`. This code triggers these to be populated. I'm not sure how it was working before with the previous code!